autopkg/autopkg_tools.py

```diff
@@ -56,6 +58,29 @@ def updated_version(self):
     def name(self):
         return self.plist["Input"]["NAME"]
 
+    def verify(self):
+        try:
+            cmd = ["/usr/local/bin/autopkg", "verify-trust-info", self.path, "-vvv"]
+            cmd = " ".join(cmd)
+            if DEBUG:
+                print("Running " + str(cmd))
+
+            p = subprocess.Popen(
+                cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True
+            )
+            (output, err) = p.communicate()
+            p_status = p.wait()
+            if p_status == 0:
+                self.verified = True
+            else:
+                err = err.decode()
+                self.results["message"] = err
+
+        except subprocess.CalledProcessError as e:
+            raise e
```

Comment:
I'm assuming `verify-trust-info` exists `0` even if trust verification fails, since that's handled above? If that's so, it's fine to expose this error. If not, should we handle verification failed events in the `except`?

Reply:
`verify-trust-info` exits `1` if the recipe fails. Did you want to simply get rid of the else statement here then?